tests/test-cluster.py

```diff
@@ -463,3 +464,119 @@ def test_no_cert_reissue_in_nodes(self):
         for vm in self.VM:
             lock_files = vm.run("ls /var/snap/microk8s/current/var/lock/")
             assert "no-cert-reissue" in lock_files.decode()
+
+
+class TestUpgradeCluster(object):
+    @pytest.fixture(autouse=True, scope="module")
+    def setup_old_versioned_cluster(self):
+        """
+        Provision VMs of the previous version and form the cluster.
+        """
+        try:
+            print("Setting up cluster of an older version")
+            track, risk_level = channel_to_test.split('/')
+            if track == "latest":
+                track = "1.26"   # TODO: remove this hardcoding
+            track = float(track) - 0.01
+            older_version = str(track) + '/' + risk_level
```

Comment:
For risk level lets stick with stable so we do not need to deal with any issues on edge

Reply:
How do we tackle track being `latest` rather than a server version?

Reply:
The latest track we should use the old stable track. For example now the current stable is 1.26, the latest track should build snaps that must cluster with the 1.25 versioned clusters. Have a look at https://github.com/canonical/microk8s/blob/master/tests/test-upgrade-path.py#L36 on how we figure out the current stable release.